pkgs/development/tools/cddl/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
```

Comment:
this should be a bundlerApp if it just exports a bin

Reply:
@fdns is it used just for the command in bin?